src/main/java/di/ConfigModule.java

```diff
@@ -25,6 +29,16 @@ public void configure(){
         /*
          * Injection of movie service
          */
```

Comment:
I will make and TODO in Trello for this since I have to look more into the "Correct" way of separating all the bindings 
for now everything just lays here and does the job 
all of them are in Singleton.class because we want to ensure all of this is instantiated only once at runtime I believe 
correct me if I am Wong